terraform/istio/install_istio.sh

```diff
@@ -0,0 +1,60 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#!/bin/bash
+
+# Install Istio on the single cluster for Stackdriver Sandbox
+
+# Download Istio
+
+echo "### "
+echo "### Begin install istio control plane"
+echo "### "
+
+# Set vars for DIRs
+ISTIO_VERSION="${ISTIO_VERSION:-1.6.2}"
+
+# Set the working directory to our current directory (/sandbox/terraform)
```

Comment:
is this comment still correct? It looks like the script is now in `/terraform/istio`

Reply:
Changed :)